ner_v2/detectors/temporal/date/hi/date_detection.py

```diff
@@ -0,0 +1,59 @@
+from chatbot_ner.config import ner_logger
+from ner_v2.detectors.temporal.constant import LANGUAGE_DATA_DIRECTORY
+from ner_v2.detectors.temporal.date.standard_regex_date import BaseRegexDate
+import datetime
+import pytz
+import os
+
+
+class DateDetector(BaseRegexDate):
+    def __init__(self, entity_name, timezone='UTC'):
+
+        self.text = ''
```

Comment:
Why are we doing all initialisations again when it already happens in BaseRegexDate? No need to do it again.